src/isar/services/utilities/json_service.py

```diff
@@ -37,3 +45,85 @@ def default(self, o):
         if isinstance(o, datetime):
             return o.isoformat()
         return super().default(o)
+
+
+class MissionDecoder(json.JSONDecoder):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(object_hook=self.object_hook, *args, **kwargs)
+
```

Comment:
Previously we have used the `from_dict()` function to perform this functionality. Have a look at the `BaseReader` class and the `dict_to_dataclass` function here: https://github.com/equinor/isar/blob/main/src/isar/services/readers/base_reader.py

The usage can be seend here in the `get_mission` function. https://github.com/equinor/isar/blob/main/src/isar/services/readers/mission_reader.py